pkg/k8s/factory_functions_test.go

```diff
@@ -1048,6 +1048,15 @@ func (s *K8sSuite) Test_ConvertToCNP(c *C) {
 				CiliumNetworkPolicy: &v2.CiliumNetworkPolicy{},
 			},
 		},
+		{
+			name: "A CCNP where it doesn't contain neither a spec nor specs",
+			args: args{
+				obj: &v2.CiliumClusterwideNetworkPolicy{},
+			},
+			want: &types.SlimCNP{
+				CiliumNetworkPolicy: &v2.CiliumNetworkPolicy{},
+			},
+		},
```

Comment:
You can either drop this change or you also import this change https://github.com/cilium/cilium/commit/c813a15041dd8996a686a218ffae61d74b4ef137#diff-e3ab607a4306e41dfa6064994ce4ca7d1bde3a7adf750133bdd018d4b80c8e55R1106